shap/utils/_general.py

```diff
@@ -302,10 +305,10 @@ def __getitem__(self, item):
         new_self._ops.append(["__getitem__", [item], {}])
         return new_self
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str) -> "OpChain":
         # Don't chain special attributes
         if name.startswith("__") and name.endswith("__"):
-            return None
+            return None  # type: ignore
```

Comment:
This appears to be a special case that was put in to avoid issues with ReadTheDocs when `inspect` is sued to lookup special attributes of objects. It doesn't seem to be invoked outside of building docs.